analyzer/src/main/kotlin/managers/utils/NuGetSupport.kt

```diff
@@ -113,32 +112,58 @@ class NuGetSupport(
 
     private val registrationsBaseUrls: List<String> = getRegistrationsBaseUrls(definitionFile)
 
-    private val packageMap = mutableMapOf<Identifier, Pair<AllPackageData, Package>>()
-
     private inline fun <reified T> ObjectMapper.readValueFromUrl(url: String): T {
         val text = client.downloadText(url).getOrThrow()
         return readValue(text)
     }
 
-    private fun getAllPackageData(id: Identifier): AllPackageData {
-        // Note: The package name in the URL is case-sensitive and must be lower-case!
-        val lowerId = id.name.lowercase()
+    private fun getAllPackageData(
+        directDependencies: Collection<Identifier>
+    ): Pair<Map<Identifier, AllPackageData>, Set<OrtIssue>> {
+        val issues = mutableSetOf<OrtIssue>()
```

Comment:
The below code seems to do significantly more than just to "Separate fetching data from mapping it". The more important change clearly is that now multiple ids are fetched at once. It's a bit inconvenient to review both changes in the same diff.

Reply:
Resolving this as this seems to be just a general remark.